File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/entity/SysRole.java
package ink.ckx.mo.system.api.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import ink.ckx.mo.common.core.base.BaseEntity;
import ink.ckx.mo.common.web.enums.StatusEnum;
import lombok.Data;

@Data
public class SysRole extends BaseEntity {

    @TableId(type = IdType.ASSIGN_ID)
    private Long id;

    private String name;

    private String code;

    private Integer sort;

    private Integer dataScope;

    private StatusEnum status;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/entity/SysRole.kt
package ink.ckx.mo.admin.api.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import ink.ckx.mo.common.core.base.BaseEntity
import ink.ckx.mo.common.web.enums.StatusEnum

data class SysRole(
    @TableId(type = IdType.ASSIGN_ID)
    var id: Long? = null,
    var name: String? = null,
    var code: String? = null,
    var sort: Int? = null,
    var dataScope: Int? = null,
    var status: StatusEnum? = null,
) : BaseEntity()